sdk/src/test/java/com/mapbox/maps/attribution/AttributionParseTest.java

```diff
@@ -295,7 +297,7 @@ public void testShortOpenStreetMapString() throws Exception {
   @Test
   public void testShortOpenStreetMapWithoutCopyrightString() throws Exception {
     AttributionParser attributionParser = new AttributionParser.Options(RuntimeEnvironment.application)
-      .withAttributionData(STREETS_ATTRIBUTION, SATELLITE_ATTRIBUTION, "blabla", "")
```

Comment:
Seems the original test it intentionally verified the blabla is skipped, wonder the reason behind it, and if we break something here. cc @tobrun

Reply:
I think earlier implementation for attribution was to only accept html tags and if there is a string literal provided, it will be skipped. But, as verified on web, it accepts string literal too (as mentioned in the internal ticket). But maybe @tobrun can verify this further.

Reply:
that is correct, the business logic was optimized for links only while text is also accepted following the spec.